### ISSUE TITLE
setup.py requires `pandas>1.3` but that's a lie

### ISSUE THREAD

**Issue by RasmusOrsoe** [graphnet-team/graphnet#622]
**Describe the bug**
GraphNeT requires `pandas>2.0` but we specify 1.3 in `setup.py`. 

Specifically, when converting i3 files to SQLite, installations with `pandas<2.0` produce the error: `TypeError: __init__() got multiple values for argument 'schema
`
**To Reproduce**
1. install graphnet
2. downgrade to any pandas version lower than 2.0
3. run  tests/data/test_dataconverters_and_datasets.py::test_dataconverter[sqlite] 
4. see error

**Full traceback**
Please include the full error message to allow for debugging
```
src/graphnet/data/utilities/parquet_to_sqlite.py:98: in run
    self._save_to_sql(
src/graphnet/data/utilities/parquet_to_sqlite.py:128: in _save_to_sql
    create_table_and_save_to_sql(
src/graphnet/data/sqlite/sqlite_utilities.py:148: in create_table_and_save_to_sql
    save_to_sql(df, table_name=table_name, database_path=database_path)
src/graphnet/data/sqlite/sqlite_utilities.py:53: in save_to_sql
    df.to_sql(table_name, con=engine, index=False, if_exists="append")
/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py:2872: in to_sql
    sql.to_sql(
/usr/local/lib/python3.8/dist-packages/pandas/io/sql.py:708: in to_sql
    pandas_sql = pandasSQL_builder(con, schema=schema)
/usr/local/lib/python3.8/dist-packages/pandas/io/sql.py:788: in pandasSQL_builder
    return SQLDatabase(con, schema=schema, meta=meta)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <pandas.io.sql.SQLDatabase object at 0x7f3cd0f67670>
engine = Engine(sqlite:////__w/graphnet/graphnet/data/tests/output/oscNext_genie_level7_v02_first_5_frames_from_parquet/data/oscNext_genie_level7_v02_first_5_frames_from_parquet.db)
schema = None, meta = None

    def __init__(self, engine, schema: str | None = None, meta=None):
        self.connectable = engine
        if not meta:
            from sqlalchemy.schema import MetaData
    
>           meta = MetaData(self.connectable, schema=schema)
E           TypeError: __init__() got multiple values for argument 'schema'
```

**Comment by AMHermansen** [comment]
Hello @RasmusOrsoe From what I can see in this traceback, it seems to be a versioning error in sqlalchemy and not pandas. Could you maybe include a list of all your packages, I suspect your version of sqlalchemy is "too new" for the older version of pandas, and an older version should be used :-)

**Comment by RasmusOrsoe** [comment]
@AMHermansen for transparency:

Jiyuan encountered this bug in an environment that Benedikt created for him in a singularity image - intended for usage on the icecube compute cluster. I tried to recreate the bug outside the cluster by using the github runners, and the traceback is unfortunately what it is. You're right that one could adjust the sqlalchemy version to remove the error too, and I attribute the error Jiyuan encountered to a version mismatch between the two packages. Closing the issue.
